.eslintrc.json

```diff
@@ -0,0 +1,29 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true,
+        "node": true
+    },
+    "extends": "airbnb-base",
```

Comment:
You should use airbnb, not airbnb-base I guess

Reply:
Fixed.